lib/dealership.rb

```diff
@@ -0,0 +1,57 @@
+class Dealership
+  attr_reader :name,
+              :address,
+              :inventory,
+              :inventory_count
+
+  def initialize(name, address)
+    @name            = name
+    @address         = address
+    @inventory       = []
+    @inventory_count = 0
```

Comment:
Another way you could have done inventory count was to make it a method rather than a read attribute that was calculated based on the @inventory

```
def inventory_count
  @inventory.count
end
```

Definitely not necessary, just another way of doing things :)